### DISCUSSION TITLE
'npx shadcn-ui@latest init' does not work with openssl issues.

### DISCUSSION THREAD

**Discussion by jonghyeonLeee** [shadcn-ui/ui#3829] [Q&A]
Hi. 

I was following the documentation(https://ui.shadcn.com/docs/cli) to set up the initialization, 
I noticed that when I run `npx shadcn-ui@latest init`, it doesn't proceed with an error message. 

`FetchError: request to https://ui.shadcn.com/registry/styles/index.json failed, reason: write EPROTO 00CC0CF901000000:error:0A000152:SSL routines:final_renegotiate:unsafe legacy renegotiation disabled:../deps/openssl/openssl/ssl/statem/extensions.c:922:
`

Has anyone else encountered this issue?

OS: mac 14.5(23F79)
NODE: v22.2.0, v18.20.3

**Comment by jonghyeonLeee** [comment]
<img width="432" alt="image" src="https://github.com/shadcn-ui/ui/assets/123723249/8dfe4adc-3bc4-4629-84bd-42b09e47e7da">

I checked that when I requested it with curl, there was no problem :(

**Reply by AiDeCoder9** [reply]
i m facing the same issue

**Reply by AiDeCoder9** [reply]
found the solution instead of npx shadcn-ui@latest init use  npx shadcn-ui init.
I dont why the command doesnt work with@latest tag. @jonghyeonLeee hope it helps you

**Comment by AiDeCoder9** [comment]
use npx shadcn-ui init instead of npx shadcn-ui@latest
